plugins/pics/info.json

```
@@ -0,0 +1,220 @@
+{
+  "name": "pics",
+  "version": "2.0.0",
+  "authors": ["boi123212321", "john4valor", "leadwolf"],
+  "description": "Find actor, scene, movie, studio images based on local files. GIF support.",
+  "pluginEvents": [
+    "actorCreated",
+    "actorCustom",
+    "sceneCreated",
+    "sceneCustom",
+    "movieCreated",
+    "studioCreated",
+    "studioCustom"
+  ],
+  "arguments": [
+    {
+      "name": "dry",
+      "type": "Boolean",
+      "required": false,
+      "default": false,
+      "description": "Whether to commit data changes"
+    },
+    {
+      "name": "actors",
+      "type": "Array",
+      "required": true,
+      "default": [],
+      "description": "Array of picture search configurations for actors"
+    },
+    {
+      "name": "actors.[0]",
+      "type": "Object",
+      "required": true,
+      "default": {},
+      "description": "One configuration for a type of actor picture"
+    },
+    {
+      "name": "actors.[0].prop",
+      "type": "`'thumbnail' \\| 'altThumbnail' \\| 'avatar' \\| 'hero' \\| 'extra'`",
+      "required": true,
+      "default": "thumbnail",
+      "description": "The type of picture that should be attached to the actor. Set to `'extra'` to add any image you want to the gallery"
+    },
+    {
+      "name": "actors.[0].path",
+      "type": "string",
+      "required": true,
+      "default": "./path/to/all/actor/pictures",
+      "description": "The path in which to search for this picture"
+    },
+    {
+      "name": "actors.[0].searchTerms",
+      "type": "string[]",
+      "required": false,
+      "default": ["thumbnail"],
+      "description": "Extra terms that the picture path should contain"
+    },
+    {
+      "name": "actors.[0].blacklistTerms",
+      "type": "string",
+      "required": false,
+      "default": [],
+      "description": "Terms that should not be found in an image path"
+    },
+    {
+      "name": "actors.[0].max ",
```

Space

---

I don't understand

---

"actors.[0].max " should be "actors.[0].max"

---

```suggestion
      "name": "actors.[0].max",
```

I forgot I could do suggestions

---

Oh ok, I thought I was crazy